addons/crm_project_issue/__openerp__.py

```diff
@@ -12,8 +13,8 @@
 Link module to map leads to issues
         """,
     'data': [
-        'project_issue_view.xml'
+        'wizard/crm_lead2projectissue_wizard_view.xml',
+        'views/crm_lead_views.xml'
```

Comment:
@rpi-odoo 
Hey man Oops :laughing: ,
I will defiantly think,
Its my great mistake,
But you really needs revise your code.
Then when you free take change of Nursery , Actually you need it.
And as per your code take only your coffee that's good.

![ping-pong-tongue-animated-frogs-breakfast](https://cloud.githubusercontent.com/assets/14940179/12096801/26d8f582-b2d5-11e5-81b9-3b1f964fbfb4.gif)